esp/src/src-react/components/FileDetails.tsx

```diff
@@ -76,7 +90,7 @@ export const FileDetails: React.FunctionComponent<FileDetailsProps> = ({
             <PivotItem headerText={nlsHPCC.Graphs} itemKey="Graphs" itemCount={file?.Graphs?.ECLGraph?.length} headerButtonProps={{ disabled: isDFUWorkunit }} style={pivotItemStyle(size, 0)}>
                 <FileDetailsGraph cluster={cluster} logicalFile={logicalFile} />
             </PivotItem>
-            <PivotItem headerText={nlsHPCC.Workunit} itemKey="Workunit" style={pivotItemStyle(size, 0)}>
+            <PivotItem headerText={nlsHPCC.Workunit} itemKey="Workunit" style={pivotItemStyle(size, 0)} headerButtonProps={!!file?.isSuperfile ? { disabled: true, style: { color: theme.palette.neutralTertiary } } : {}}>
```

Comment:
@GordonSmith This isn't very noticeable perhaps, but I'm not sure why this color (theme.palette.neutralTertiary) is off. It seems to be defined as #d1d1d1 in src/src-react/themes.ts, which is the same color that is used for Buttons when disabled. But used here, the PivotItem's button is getting a text color of #a19f9d. Have you noticed any problems with colors like this elsewhere?

Reply:
I have noticed that disabled tabs aren't very pronounced.  I will take a look